Library/Formula/datomic.rb

```diff
@@ -1,9 +1,9 @@
 class Datomic < Formula
   desc "Database separating transactions, storage and queries"
   homepage "http://www.datomic.com/"
-  url "https://my.datomic.com/downloads/free/0.9.5206"
-  version "0.9.5206"
-  sha256 "b5b406d301b3e3fd4ff61d3e20946fb67fc5adae269bc2ee3bbca0819e02de3c"
+  version "0.9.5302"
+  url "https://my.datomic.com/downloads/free/#{version}"
```

Comment:
Sorry, didn't know! (It's quite common in homebrew-cask.) Maybe this rule should be mentioned in the formula cookbook?